embedding/cpp/flutter_app.cc

```diff
@@ -11,30 +11,9 @@
 
 bool FlutterApp::OnCreate() {
   TizenLog::Debug("Launching a Flutter application...");
-
-  // Read engine arguments passed from the tool.
-  Utils::ParseEngineArgs(&engine_args_);
-
-  std::vector<const char *> switches;
-  for (auto &arg : engine_args_) {
-    switches.push_back(arg.c_str());
-  }
-  std::vector<const char *> entrypoint_args;
-  for (auto &arg : dart_entrypoint_args_) {
-    entrypoint_args.push_back(arg.c_str());
-  }
-
-  FlutterDesktopEngineProperties engine_prop = {};
-  engine_prop.assets_path = "../res/flutter_assets";
-  engine_prop.icu_data_path = "../res/icudtl.dat";
-  engine_prop.aot_library_path = "../lib/libapp.so";
-  engine_prop.switches = switches.data();
-  engine_prop.switches_count = switches.size();
-  engine_prop.entrypoint = dart_entrypoint_.c_str();
-  engine_prop.dart_entrypoint_argc = entrypoint_args.size();
-  engine_prop.dart_entrypoint_argv = entrypoint_args.data();
-
-  engine_ = FlutterDesktopEngineCreate(engine_prop);
+  engine_ = FlutterEngine::Create("../res/flutter_assets", "../res/icudtl.dat",
+                                  "../lib/libapp.so", dart_entrypoint_,
+                                  dart_entrypoint_args_);
```

Comment:
dart_entrypoint_ and dart_entrypoint_args_ in FlutterApp class are currently not set anywhere
Since getting Args from ParseEngineArgs is already implemented in FlutterEngine, it is better to delete the member variable and this argument before adding a setter getter for a separate entrypoint.

Reply:
Thank you for review, here is my answer.

> dart_entrypoint_ and dart_entrypoint_args_ in FlutterApp class are currently not set anywhere

`FlutterApp` and `FlutterService` provide them as protected members. The host implementations(including tempate) that inherit from these base classes(`FlutterApp` or `FlutterService`) can use them directly such as window offset.

> Since getting Args from ParseEngineArgs is already implemented in FlutterEngine,

The engine arguments is delivered by the tool(flutter-tizen), but not the dart arguments. so it's better to be presented as a programmable interface as it is now.

> this argument before adding a setter getter for a separate entrypoint.

I think It is better to deliver them when you create them because they correspond to the current CAPI.